sdk/core/core-rest-pipeline/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.1.0-beta.3 (Unreleased)
+## 1.1.0 (2021-06-04)
```

Comment:
I thought we didn't like to release on Fridays?

Reply:
I’ll move it to Thursday!